db_assessment/import_db_assessment.py

```diff
@@ -518,20 +533,25 @@ def importCSVToBQ(gcpProjectName,bqDataset,tableName,fileName,skipLeadingRows,au
             print ('\n FAILED: Optimus Prime could not import the filename "{}" into "{}" because of the error "{}".\n'.format(fileName,table_id,importErr))
 
             print ('   Table Schema = {}'.format(schema))
-            return False
+            if 'csv' not in fileName :
```

Comment:
Do we really need this?

Reply:
It is removed in latest commit for bug132 branch